ModularTegustation/tegu_items/associations/augment_components.dm

```diff
@@ -0,0 +1,1261 @@
+	human_parent.physiology.pale_mod -= total_damage_resist
+	to_chat(human_parent, span_nicegreen("You took [total_damage_resist]% less damage! Due to Reinforcement Nanties"))
+
+/datum/component/augment/reinforcement_nanties/after_take_damage_effect(datum/source, damage, damagetype, def_zone)
+	. = ..()
+	human_parent.physiology.red_mod += total_damage_resist
+	human_parent.physiology.white_mod += total_damage_resist
+	human_parent.physiology.black_mod += total_damage_resist
+	human_parent.physiology.pale_mod += total_damage_resist
+	total_damage_resist = 0
+	damage_resist_mult = 0
+
+//Stalwart Form
+/datum/component/augment/stalwart_form/Initialize()
+	. = ..()
+	human_parent.physiology.stun_mod -= 0.9
+	human_parent.physiology.red_mod += 0.15
+	human_parent.physiology.black_mod += 0.15
+
+/datum/component/augment/stalwart_form/Destroy()
+	. = ..()
+	human_parent.physiology.stun_mod += 0.9
+	human_parent.physiology.red_mod -= 0.15
+	human_parent.physiology.black_mod -= 0.15
+
+///Status
+
+//Double check if vigors work
+//Make augments work with gauntlets
+//Time Mortarium and Reflictive Tremor
+
+//Burn Vigor
+/datum/component/augment/burn_vigor
+	var/damage_buff = 10
+	var/total_damage_buff = 0
+
+/datum/component/augment/burn_vigor/attack_effect(datum/source, mob/living/target, mob/living/user, obj/item/item)
+	. = ..()
+	var/datum/status_effect/stacking/lc_burn/UB = user.has_status_effect(/datum/status_effect/stacking/lc_burn)
+	if(UB)
+		total_damage_buff = round(UB.stacks/5) * damage_buff * repeat
+	else
+		total_damage_buff = 0
+	human_parent.extra_damage += total_damage_buff
+	to_chat(human_parent, span_nicegreen("You dealt [total_damage_buff]% more damage! Due to Burn Vigor"))
+
+/datum/component/augment/burn_vigor/afterattack_effect(datum/source, atom/target, mob/user, proximity_flag, obj/item/item)
+	. = ..()
+	human_parent.extra_damage -= total_damage_buff
+	total_damage_buff = 0
+
+//Burn Vigor
+/datum/component/augment/bleed_vigor
+	var/damage_buff = 10
+	var/total_damage_buff = 0
+
+/datum/component/augment/bleed_vigor/attack_effect(datum/source, mob/living/target, mob/living/user, obj/item/item)
+	. = ..()
+	var/datum/status_effect/stacking/lc_bleed/UB = user.has_status_effect(/datum/status_effect/stacking/lc_bleed)
+	if(UB)
+		total_damage_buff = round(UB.stacks/5) * damage_buff * repeat
+	else
+		total_damage_buff = 0
+	human_parent.extra_damage += total_damage_buff
+	to_chat(human_parent, span_nicegreen("You dealt [total_damage_buff]% more damage! Due to Bleed Vigor"))
+
+/datum/component/augment/bleed_vigor/afterattack_effect(datum/source, atom/target, mob/user, proximity_flag, obj/item/item)
+	. = ..()
+	human_parent.extra_damage -= total_damage_buff
+	total_damage_buff = 0
+
+//Tremor Defense
+/datum/component/augment/tremor_defense
+	var/damage_resist = 10
+	var/total_damage_resist = 0
+
+/datum/component/augment/struggling_defense/take_damage_effect(datum/source, damage, damagetype, def_zone)
```

Comment:
Minor typo, should be `tremor_defense` here and a lines below, not `struggling_defense`.